pom.xml

```diff
@@ -241,6 +241,13 @@
       <artifactId>jackson-dataformat-cbor</artifactId>
       <version>${jackson.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>javax.json</groupId>
+      <artifactId>javax.json-api</artifactId>
+      <version>1.0</version>
+    </dependency>
+
```

Comment:
Given that we've already got jackson in the project can we avoid another json library?

Reply:
I imagine so... I'll review.